docs/docusaurus/docs/reference/learn/data_quality_use_cases/distribution_resources/distribution_expectations.py

```diff
@@ -0,0 +1,146 @@
+
+2. Run the following command from the repo root dir in a second terminal:
+pytest --postgresql --docs-tests -k "data_quality_use_case_distribution_expectations" tests/integration/test_script_runner.py
+"""
+
+# This section loads sample data to use for CI testing of the script.
+import pathlib
+
+import great_expectations as gx
+import great_expectations.expectations as gxe
+from tests.test_utils import load_data_into_test_database
+
+CONNECTION_STRING = "postgresql+psycopg2://postgres:@localhost/test_ci"
+
+GX_ROOT_DIR = pathlib.Path(gx.__file__).parent.parent
+
+# Add test data to database for testing.
+load_data_into_test_database(
+    table_name="purchases",
+    csv_path=str(
+        GX_ROOT_DIR
+        / "tests/test_sets/learn_data_quality_use_cases/distribution_purchases.csv"
+    ),
+    connection_string=CONNECTION_STRING,
+)
+
+context = gx.get_context()
+
+datasource = context.data_sources.add_postgres(
+    "postgres database", connection_string=CONNECTION_STRING
+)
+
+data_asset = datasource.add_table_asset(name="data asset", table_name="purchases")
+batch_definition = data_asset.add_batch_definition_whole_table("batch definition")
+batch = batch_definition.get_batch()
+
+suite = context.suites.add(gx.ExpectationSuite(name="example distribution expectations"))
+
+#############################
+# Start Expectation snippets.
+
+suite.add_expectation(
+    # <snippet name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/distribution_resources/distribution_expectations.py ExpectColumnQuantileValuesToBeBetween">
+    gxe.ExpectColumnQuantileValuesToBeBetween(
+        column="purchase_amount",
+        quantile_ranges={
+            "quantiles": [0.5, 0.9],
+            "value_ranges": [
+                [50, 200],
+                [500, 2000]
+            ]
+        }
+    )
+    # </snippet>
+)
+
+suite.add_expectation(
+    # <snippet name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/distribution_resources/distribution_expectations.py ExpectColumnStdevToBeBetween">
+    gxe.ExpectColumnStdevToBeBetween(
+        column="purchase_amount",
+        min_value=500,
+        max_value=2000
+    )
+    # </snippet>
+)
+
+suite.add_expectation(
+    # <snippet name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/distribution_resources/distribution_expectations.py ExpectColumnValuesToBeBetween">
+    gxe.ExpectColumnValuesToBeBetween(
+        column="product_rating",
+        min_value=1,
+        max_value=5,
+    )
+    # </snippet>
+)
+
+suite.add_expectation(
+    # <snippet name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/distribution_resources/distribution_expectations.py ExpectColumnValueZScoresToBeLessThan">
+    gxe.ExpectColumnValueZScoresToBeLessThan(
+        column="purchase_amount",
+        threshold=3,
+    )
+    # </snippet>
+)
+
+suite.add_expectation(
+    # TODO: Paul is not sure how to add multiple expectations to the suite at once, or some way to
```

Comment:
I'm going to take a look at this tomorrow and see if there's a way to set this up so that the snippet doesn't have the extra character. We may have to assign the two Expectations to named variables and then add the variables using  `add_expectation(...)` outside of the snippet.

FYI, I checked in with Engineering on this topic, and currently, there's not a method that enables you to add multiple Expectations at once. I've generally been using a list comprehension or loop to do it.